modules/40_techpol/NPi2018/input/files

```diff
@@ -1,3 +1 @@
-f40_NDC.cs3r
```

Comment:
How it this related the the new CES parameters?

Reply:
independent, only house keeping, old file which is no longer generated nor used